packages/cli/src/metadata/handler.ts

```diff
@@ -81,7 +83,13 @@ const metadataHandler = async (options: MetadataOpts, logger: Logger) => {
   }
 
   try {
-    // Import the adaptor
+    //Install and import the adaptor
+    const autoInstall = getAutoinstall(adaptor);
+    if (autoInstall) {
+      logger.info('Auto-installing language adaptors');
```

Comment:
This log line is redundant, the install function does all the logging we need.

For the record, here's the actual installl output:
```
[CLI] ℹ Auto-installing language adaptors
[CLI] ✔ Installing packages...
[CLI] ℹ Looked up latest version of @openfn/language-salesforce: found 4.6.2
[CLI] ℹ Will install @openfn/language-salesforce version 4.6.2
[CLI] ✔ Installed @openfn/language-salesforce@4.6.2
[CLI] ✔ Installation complete in 10.526s
```